nilearn/plotting/img_plotting.py

```diff
@@ -313,6 +314,78 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
 # Usage-specific functions
 
 
+def plot_4d(img, anat_img=MNI152TEMPLATE, cut_coords=None,
```

Comment:
The name of the function is a bit misleading. Given that EPI is 4d, I may want to put it in this function, which is not the intended purpose. Something like `plot_atlas` or `plot_contours` would be more meaningful I think.